ansys/dpf/core/elements.py

```diff
@@ -118,11 +135,12 @@ def __str__(self):
 
     @property
     def type(self) -> int:
-        """Element type of this element.
+        """Type of the element.
         
         Returns
         -------
-        element_type : element_types
+        int
```

Comment:
@PipKat this one is not an int, it's a :class:`ansys.dpf.core.elements.element_types`